ethereum/smart-contract/contracts/interfaces/IGatewayNetwork.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.19;
+
+abstract contract  IGatewayNetwork {
+    // Ranges from 0% - 100%
+    /**
+     * @dev Struct that describes the fees of each 
+     */
+    struct NetworkFeesPercentage {
+        // Token address used to pay fees. Zero address for this mean native eth
+        address tokenAddress;
+        uint16 issueFee;
+        uint16 refreshFee;
+        uint16 expireFee;
+    }
+
+    /**
+     * @dev This struct represents data associated with the state of a gatekeeper network
+     */
+    struct GatekeeperNetworkData {
+        // the authority for this network
+        address primaryAuthority;
+
+        // Unique identifier for a network.
+        bytes32 name;
+        
+        // The default expiration time of passes on this network in seconds
+        uint passExpireTimeInSeconds;
+        
+        //Features on the network, index relates to which feature it is.
+        uint32 networkFeatures;
```

Comment:
Add 1 feature (pass expiration based on the feature in the gateway token contract)